app/routes/faq.tsx

```diff
@@ -34,6 +57,18 @@ export default function FAQ() {
 				. Neat!
 			</Aside>
 
+			<Heading level={3}>Is there a dress code?</Heading>
+			<BodyText>
+				Casual street clothing. Most of us wear t-shirts and short/jeans. We
+				intentionally do not have a business casual or formal attire vibe.
```

Comment:
Could someone reading this think they can't come straight from work if they happen to work in business casual clothing e.g. SWEs at Fidelity/etc.? I might be overthinking it in which case "No" is a perfectly valid answer :D

Reply:
Oh that's a good point. Switched to:

> We do not require business casual attire, but you can wear it if you prefer.

WDYT?